---
description: Compilerfehler CS0724
title: Compilerfehler CS0724
ms.date: 07/20/2015
f1_keywords:
- CS0724
helpviewer_keywords:
- CS0724
ms.assetid: bcdb2017-7a43-4242-b4e2-a1ae03d6d73f
ms.openlocfilehash: 88510a423e47ee4eb3a19eca1b4911a451d6513a
ms.sourcegitcommit: d579fb5e4b46745fd0f1f8874c94c6469ce58604
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 08/30/2020
ms.locfileid: "89119754"
---
# <a name="compiler-error-cs0724"></a>Compilerfehler CS0724

In einer finally-Klausel, die in der nächsten einschließenden catch-Klausel geschachtelt ist, ist keine throw-Anweisung ohne Argumente zulässig.

## <a name="example"></a>Beispiel

Im folgenden Beispiel wird CS0724 aufgrund der- `throw` Anweisung innerhalb des- `finally` klauselblocks generiert:

```csharp  
// CS0724.cs  
using System;  
  
class Program
{
    static void Test()
    {
        try
        {
            throw new Exception();
        }
        catch
        {
            try
            {
            }
            finally
            {
                throw; // CS0724
            }
        }
    }

    static void Main()
    {
    }
}
```
